src/features/creators/templates/newPackageTemplate/.github/copilot-instructions.md

```diff
@@ -0,0 +1,10 @@
+# Copilot Instructions for package_name
```

Comment:
still need to change this to add it to the root .github folder but question in the meantime @karthiknadig:

- I have it right now that the template as a folder called `<my_package>-folder` which then has a folder `<my_package>.` Did this so I had somewhere to store the `pyproject.toml` and `dev-requirements.txt`. This does make it more complicated to do an editable install of the package at the root, not sure if this is worth the concern? I also worry people will delete / more that folder with higher frequency which then makes these instructions out of date